.code-samples.meilisearch.yaml

```diff
@@ -0,0 +1,236 @@
+get_one_index_1: |-
+  $ curl \
+    -X GET 'http://localhost:7700/indexes/movies'
+list_all_indexes_1: |1-
+  $ curl \
+    -X GET 'http://localhost:7700/indexes'
+create_an_index_1: |-
+  $ curl \
+    -X POST 'http://localhost:7700/indexes' \
+    --data '{
+      "uid": "movies",
+      "primaryKey": "movie_id"
+    }'
+update_an_index_1: |-
+  $ curl \
+    -X PUT 'http://localhost:7700/indexes/movie_review' \
```

Comment:
Why not using `movies` as usually here?

Reply:
It seems that the example and the response are using `movie_review`

```javascript
$ curl \
  -X PUT 'http://localhost:7700/indexes/movie_review' \
  --data '{
      "primaryKey" : "movie_review_id"
  }'
```

```json
{
  "uid": "movie_review",
  "primaryKey": "movie_review_id",
  "createdAt": "2019-11-20T09:40:33.711324Z",
  "updatedAt": "2019-11-20T10:16:42.761858Z"
}
```

I do not remember when that example was added but if I think it was chosen because it changes the primary key to something new. The `movie` dataset does not need the primary key to be changed. By providing an other index we avoid someone copy-pasting a breaking code if they are using the `movie` index with the `movie` dataset.

What do you think?

Reply:
I think we should adapt the example to the response for now :)